### PR TITLE
Update Services “intro”

### PR DESCRIPTION
**Pull request by onelittlebecca**
Automatically generated by Netlify CMS

😎  [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/cms/services/intro/docs/services/intro/)

## Security Considerations
None

### CODE REVIEW

**Review by mheadd — DISMISSED** [review]
Looks good!

File: _docs/services/intro.md [hunk 1]
```diff
@@ -1,32 +1,56 @@
...
+```
+Or you can pass in your username and password in one line with:
+```bash
+cf cups <name-of-service> -p '{"username":"admin","password":"pa55woRD"}'
+```
+
 ## Provisioning managed services through the marketplace
 
-cloud.gov offers a marketplace of FedRAMP-authorized [managed services]({{ site.baseurl }}{% link _docs/deployment/managed-services.md %}) that we operate in a secure and compliant manner on your behalf. You can also [extend the marketplace](#extending-the-marketplace) to include additional services run by other organizations.
+cloud.gov offers a marketplace of FedRAMP-authorized \[managed services]({{ site.baseurl }}{% link _docs/deployment/managed-services.md %}) that we operate in a secure and compliant manner on your behalf. You can also [extend the marketplace](#extending-the-marketplace) to include additional services run by other organizations.
```
Comment (ccostino) [inline _docs/services/intro.md]: Is this an extraneous backslash that made its way in, or is it needed?

**Review by ccostino — COMMENTED** [review]
Thank you, @onelittlebecca!  The updates look great, thanks for taking the time to document the extra information! 😃 

One quick question - I noticed an extra backslash, is that required as a part of the updates for formatting, or no?

File: _docs/services/intro.md (hunk 1 above)
Comment (ccostino) [reply _docs/services/intro.md]: Turns out it was extraneous and causing the link to not be rendered!

**Review by ccostino — APPROVED** [review]
Thanks again, @onelittlebecca!

**Pull request merged**